core/jobs/batch_jobs/rejecting_suggestion_for_invalid_content_ids_jobs.py

```diff
@@ -80,9 +81,20 @@ def _update_suggestion_model(
             if not suggestion_change['content_id'] in total_content_ids:
                 suggestion.status = suggestion_models.STATUS_REJECTED
 
-            suggestion_change['translation_html'] = (
-                exp_domain.Exploration.fix_content(
-                    suggestion_change['translation_html']))
+            translation_html = suggestion_change['translation_html']
+            resulting_translation = []
+            if isinstance(translation_html, list):
+                for translation in translation_html:
+                    resulting_translation.append(
```

Comment:
@lkbhitesh07 We don't have any list(html) data in translation, right? ([ref](https://github.com/oppia/oppia/blob/develop/core/domain/state_domain.py#L2956))

So do we need to run fi_content on normal/unicode strings?

Reply:
Hey!
The fix_content only fixes the RTE `tags` and the content for which the suggestion is created is already been migrated or fixed. If that's what you mean.
Thanks

Reply:
@lkbhitesh07 `translation_html` can either be "HTML(String)" or "List(normalString/UnicodeString/NonHtmlString)", so I wanted to check whether it's needed to pass normal/UnicodeString into the fix_content function.

Reply:
If that is the case then we can avoid doing the changes to the list, although nothing will be changed as the function will not be able to find any RTE tags and will not make any changes.
Thanks